hikari/api/shard.py

```diff
@@ -151,10 +151,6 @@ async def join(self) -> None:
         websocket runtime itself.
         """
 
-    @abc.abstractmethod
-    async def start(self) -> None:
-        """Start the shard, wait for it to become ready."""
-
```

Comment:
Why was this removed?

Reply:
Because it's implementation detail, the only place where a sub-component will be started is in a place where internal detail is being used in-order to initialise and start it, like within bot app. Similar to how the rest interface doesn't have a start method but some implementation may require that it is explicitly "started"

Reply:
It is not. `start` and `join` can be used instead of `run` if you want to have more control over what it does.

Reply:
not a single interface in hikari.api defines a run method(?)

Reply:
> It is not. start and join can be used instead of run if you want to have more control over what it does.

and how is this relevant to the public abstract interface for a Shard instance?

Reply:
You kinda need to call `start` before `join` to start the shard :thinking:

Reply:
and the manager won't be calling it based on the abc since it'd just built the implementation class

Reply:
> You kinda need to call start before join to start the shard 🤔

also what if it's a shard implementation which just wraps around a receiver and never needs to be started itself